.github/workflows/deployStg.yml

```diff
@@ -107,15 +107,18 @@ jobs:
         with:
           client-tarball: client.tgz
           deploy-env: ${{env.DEPLOY_ENV}}
-  verify-release:
-    runs-on: ubuntu-latest
+  e2e:
     needs: [deploy]
-    defaults:
-      run:
-        working-directory: ./ops
+    if: "!contains(github.event.commits[0].message, '[skip e2e stg]')"
```

Comment:
Nice addition! Great to have in case we need to push a fix out urgently

Reply:
I'm not so sure I agree with this. If we need to make an urgent repair on something, we can theoretically leverage the Azure deployment slots to do so. Introducing a bypass to a check makes me feel squidgy.

Is there a situation we can think of where a slot-based rollback wouldn't solve an immediate problem, and let us get a fix out with all the proper checks in place?

Reply:
TBH, I don't love bypasses like this either. However, I know we've had issues with fixes taking more time than seems comfortable, and this felt like a generally low-risk job to have this sort of bypass on, which is why I didn't push back.

Overall, I'd love to audit and work to optimize our pipeline instead of adding checks like this.

Reply:
After chatting with Rin, I'm going to remove this skip option. We should strive to keep the integrity of our pipelines intact and not skip verification steps. We also talked about prioritizing work to make our pipelines more efficient to allow for faster deploys, and we have at least one ticket coming up that would do this.